src/settings.js

```diff
@@ -0,0 +1,8 @@
+export const SELECT_ICON = 'selectIcon';
```

Comment:
I'd continue the naming scheme just like mutation types etc:
export const SELECT_ICON = ''DREIPOL_VUE_UI__SELECT_ICON';

I know, i know, it's tinfoil-hatty

Reply:
i have update it thanks